File: apprefurbished/src/main/java/com/androidcollider/easyfin/managers/ui/hide_touch_outside/HideTouchOutsideModule.java
package com.androidcollider.easyfin.managers.ui.hide_touch_outside;

import dagger.Module;
import dagger.Provides;

/**
 * @author Ihor Bilous
 */

@Module
public class HideTouchOutsideModule {

    @Provides
    HideTouchOutsideManager provideHideTouchOutsideManager() {
        return new HideTouchOutsideManager();
    }
}

File: apprefurbished/src/main/java/com/androidcollider/easyfin/common/managers/ui/hide_touch_outside/HideTouchOutsideModule.kt
package com.androidcollider.easyfin.common.managers.ui.hide_touch_outside

import dagger.Module
import dagger.Provides

/**
 * @author Ihor Bilous
 */
@Module
class HideTouchOutsideModule {
    @Provides
    fun provideHideTouchOutsideManager(): HideTouchOutsideManager {
        return HideTouchOutsideManager()
    }
}